osv/malicious/pypi/mstplotlib/MAL-2025-122.json

```diff
@@ -0,0 +1,44 @@
+{
+  "modified": "2025-02-20T06:37:06Z",
+  "published": "2025-02-20T06:37:06Z",
+  "schema_version": "1.5.0",
+  "id": "MAL-2025-122",
```

Comment:
This can be left blank

Reply:
Fixed in [e00efd7](https://github.com/ossf/malicious-packages/pull/841/commits/e00efd7d777276201c7fca5eb86521c00a3f1c74).